providers/mdxProvider.tsx

```diff
@@ -3,20 +3,12 @@ import { MDXProvider as BaseMDXProvider } from '@mdx-js/react';
 import { MDXRemote } from 'next-mdx-remote';
 import HighlightJS from 'highlight.js/lib/core';
 import HighlightJavaScript from 'highlight.js/lib/languages/javascript';
-import AnchoredHeading from '@/components/AnchoredHeading';
 import NodeApiVersionLinks from '@/components/Docs/NodeApiVersionLinks';
 import type { FC } from 'react';
 import type { MDXComponents } from 'mdx/types';
 
 const mdxComponents: MDXComponents = {
   NodeApiVersionLinks: NodeApiVersionLinks,
-  h1: props => <AnchoredHeading level={1} {...props} />,
-  h2: props => <AnchoredHeading level={2} {...props} />,
-  h3: props => <AnchoredHeading level={3} {...props} />,
-  h4: props => <AnchoredHeading level={4} {...props} />,
-  h5: props => <AnchoredHeading level={5} {...props} />,
-  h6: props => <AnchoredHeading level={6} {...props} />,
-  blockquote: ({ children }) => <div className="highlight-box">{children}</div>,
```

Comment:
You shouldn't have removed the `blockquote` one, only the h1 to h6 components.

Reply:
```className="highlight-box" is already written in md files```

Reply:
Which md files? Still please do not remove this. This is not part of the scope of this PR 🙇

Reply:
Friendly reminder to add the `blockquote` entry back.

Even if there are Markdown files already with this classname, the "blockquote" conversion is used for Markdown tags not HTML ones.